.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,20 @@
+name: RuboCop
+
+on: [push, pull_request]
+
+jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby 2.7
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - name: Install gems
+        run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+      - name: Run RuboCop
+        run: bundle exec rubocop lib --parallel
```

Comment:
Moved the Rubocop check to Github Actions.  Pretty simple just install Ruby, bundle install required Gems, and then run the rubocop rake task.